test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -33,20 +31,20 @@
     String truststore = "";
     String truststorePasswordFile = "";
 
-    String implementation = "local";
+    ChannelImplementation implementation = ChannelImplementation.LOCAL;
 
-    String cacheSizeHeapRatio = "0.5";
+    double cacheSizeHeapRatio = 0.5;
     String address = "test";
     int port = 9000;
-    String seqCache = "1000";
-    String logSizeLimitPercentage = "100.0";
-    String batchSize = "100";
+    int seqCache = 1000;
+    double logSizeLimitPercentage = 100.0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651442)